src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -185,6 +185,14 @@ public int LastInsertRowId()
             => (int)raw.sqlite3_last_insert_rowid(_handle);
 
         public Stream ReadBlob(string dataTableName, string dataColumnName, long rowId)
```

Comment:
does this overload need to exist?  who is calling into ReadBlob with a string for the table name or column name?

Reply:
Well, it was public API, so not sure if it's safe to completely remove it (although the class is internal).

Reply:
I can remove this overload if we don't need it (it's currently not used anywhere)

Reply:
Yes.  Please remove the overload.  This type is completely internal to roslyn.  We do not have to worry about breaking changes here.